spec/permissions.json

```diff
@@ -0,0 +1,74 @@
+{
+  "groups": [
+    {
+      "name": "super user",
+      "id": 0,
+      "permissions": {
+        "/**": {
+          "permissions": [
+            "Create",
+            "Read",
+            "Update",
+            "Delete"
+          ],
+          "query_parameters": {}
+        }
+      }
+    },
+    {
+      "name": "full access to the default namespace",
+      "id": 499,
+      "permissions": {
+        "/**": {
+          "permissions": [
+            "Create",
+            "Read",
+            "Update",
+            "Delete"
+          ],
+          "query_parameters": {
+            "namespace": [
+              "default-namespace"
+            ]
+          }
+        }
+      }
+    },
+    {
+      "name": "full access to Jim Oliver's namespace",
+      "id": 1000,
+      "permissions": {
+        "/**": {
+          "permissions": [
+            "Create",
+            "Read",
+            "Update",
+            "Delete"
+          ],
+          "query_parameters": {
+            "namespace": [
+              "jim-oliver"
+            ]
+          }
+        }
+      }
+    }
+  ],
+  "users": [
+    {
+      "api_key_hash": "400$8$1d$ab43d035$377826aeb74a6e71637af146738c657662ac8e0b483b3d576a2492c1241f2fee",
+      "group_ids": [
+        0
+      ],
+      "name": "Administrator"
+    },
+    {
+      "api_key_hash": "400$8$1e$10ba7fc8$0c8b2dabfd44ddb779d152d8cca0e4c7a695e85013a6a2f07c4c48b235986d85",
+      "group_ids": [
+        499,
+        1000
+      ],
+      "name": "Jim Oliver"
+    }
+  ]
+}
```

Comment:
?

Reply:
wtf...?

Reply:
There is an exception being thrown when the tests are run, this file doesn't get cleaned up because of that exception

Reply:
Why it is written there? It should be in the temp directory

Reply:
That's a very good question....

Reply:
That's because of `Fixtures::PERMISSION_FILE`, it should use `File.tempname("_temp_dppm_permisions_file")`

Reply:
No, because the filename "permissions.json" is hardcoded, the server CLI only gets the directory that it's supposed to be stored in. In the case of the specs, that's `__DIR__`, which resolves to `$PROJECT_DIR/spec`.

Reply:
Not a problem, all the `data_dir: __DIR__` in `cli_spec` can be changed to the `Fixtures` path.

Reply:
So... perhaps, instead of `__DIR__` and `Fixtures::PERMISSION_FILE`, we could have `Fixtures::DIR`?